src/AdminConsole/Components/Shared/ApexCharts/Models/ApexChartOptions.cs

```diff
@@ -12,6 +12,8 @@ public class ApexChartOptions<TXValueType, TYValueType>
 
     public Xaxis<TXValueType>? Xaxis { get; set; }
```

Comment:
I know this didn't change but `Xaxis` should probably be `XAxis`

Reply:
[No, it would mess with the serialization: https://apexcharts.com/docs/formatting-axes-labels/](https://apexcharts.com/docs/formatting-axes-labels/)